include/deal.II/base/polynomial.h

```diff
@@ -87,7 +95,8 @@ namespace Polynomials
      * Return the value of this polynomial at the given point.
      *
      * This function uses the Horner scheme for numerical stability of the
-     * evaluation.
+     * evaluation for polynomials in the coefficient form or the product of
+     * terms involving the roots if that representation is used.
```

Comment:
I see. If you think that this is obvious to the reader from the class documentation (which I didn't see as part of this patch, nor take the time to look at), then this is ok with me.